readerwriterqueue.h

```diff
@@ -220,6 +226,14 @@ class ReaderWriterQueue
 		return inner_enqueue<CannotAlloc>(std::forward<T>(element));
 	}
 
+#if MOODYCAMEL_HAS_EMPLACE
+	// Like try_enqueue() but emplace semantics (ie construct-in-place)
```

Comment:
Nitpick: Missing `with` between `but` and `emplace`. Also, `i.e.` needs periods.

Reply:
Done